src/App.tsx

```diff
@@ -77,6 +141,27 @@ const App = () => {
         });
         if (enable2FA && !authenticatorSecret) history.push('/register/2fa');
         else if (enable2FA && !isSecondFactorAuthenticated) history.push('/2fa');
+        appDispatch({ type: 'loading' });
+        return (asyncGetRequest(makeAPIPath('/blocks')));
+      })
+      .then(({ data }: { data: RawUserInfoType[] }) => {
+        const list = data.map((user) => user.name);
+        appDispatch({ type: 'renewBlockList', list });
+        return (asyncGetRequest(makeAPIPath('/channels/me')));
+      })
+      .then(({ data }) => {
+        setChannels(data.map((channel: RawChannelType) => makeChannelInfo(channel)));
+        return (asyncGetRequest(makeAPIPath('/dmers')));
+      })
+      .then(({ data }) => {
+        setDMs(data.map((dm: RawUserInfoType): DMRoomType => ({
+          id: dm.id,
+          name: dm.name,
+          avatar: makeAPIPath(`/${dm.avatar}`),
+          status: dm.status,
+          latestMessage: DM_INIT,
+          unreads: 0,
+        })));
       })
       .catch((error) => {
         if (error.response) {
```

Comment:
```suggestion
        if (error.response) userDispatch({ type: 'reset' });
        else toast.error(error.message);
      });
```
저는 어떤 게 좋을 지는 모르겠는 데, App.tsx에서 이 코드를 이렇게 바꾸는 게 나을 까요? 아니면 그대로 두는 게 나을 까요?

Reply:
음 이 부분은 로직 문제가 아니고 코드 스타일 문제라 딱 안 와닿네요! 이렇게 쓰여있는 부분이 신경쓰이신다면 if문 내용물이 한 줄인 코드를 찾아서 전체적으로 리팩토링 해주는 게 좋을 것 같고, 이 줄 스타일 수정을 위해 커밋 하나 더 쌓는 건 크게 의미가 없을 것 같아요.
그리고 이런 식으로 여러 줄에 걸친 코드 리뷰하시는 경우에는 여러 줄 전체 블럭잡아서 suggestion 보내주시면 알아보기 더 편할 것 같습니다 ㅎㅎ